engines/voyeur/data.cpp

```diff
@@ -152,7 +151,6 @@ void SVoy::synchronize(Common::Serializer &s) {
 	s.syncAsSint16LE(_fadingStep1);
 	s.syncAsSint16LE(_fadingStep2);
 	s.syncAsSint16LE(_fadingType);
-	s.syncAsSint16LE(_victimNumber);
```

Comment:
Probably this should be something like the following not to break compatibility with the old saves:
```c++
s.skip(sizeof(int16), 0, 2);
```

Reply:
Good idea, I've sone exactly that.